site/src/css/custom.css

```diff
@@ -7,6 +7,15 @@
 @import url('https://fonts.googleapis.com/css2?family=Overpass:wght@200&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Overpass+Mono:wght@300&display=swap');
 
+/* latin */
+@font-face {
+  font-family: 'Overpass-Bold';
+  font-style: normal;
+  font-weight: 800;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/overpass/v5/qFdA35WCmI96Ajtm81kCdc7N4ho.woff2) format('woff2');
```

Comment:
Rather than linking to the static reference I believe you could do something like the above import statements to keep it consistent with the existing. This should let you remove a few lines here too.
`@import url('https://fonts.googleapis.com/css2?family=Overpass:wght@800&display=swap');`